<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BARCoMmS_CFDP</class>
 <widget class="QMainWindow" name="BARCoMmS_CFDP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>721</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1071</width>
    <height>721</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1071</width>
    <height>721</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CFDP - BARCoMmS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_helpAndExit">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>530</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Help/Exit</string>
    </property>
    <widget class="QPushButton" name="pushButton_help">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_exit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_output">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>751</width>
      <height>551</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
    <property name="maximumBlockCount">
     <number>500</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_requests">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Requests</string>
    </property>
    <widget class="QPushButton" name="pushButton_abandon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Abandon</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_report">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Report</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_resume">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Resume</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_suspend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Suspend</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_put">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Put</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_engineToggles">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>60</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Engine Toggles</string>
    </property>
    <widget class="QPushButton" name="pushButton_disableEngineMessages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable
Engine
Messages</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_freeze">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Freeze
Engine</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_misc">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>400</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Misc.</string>
    </property>
    <widget class="QPushButton" name="pushButton_robust">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Robustness
Test</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_echo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Echo</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>630</y>
      <width>751</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_statuses">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>121</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Statuses</string>
    </property>
    <widget class="QPushButton" name="pushButton_transStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Trans
Status</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_detailedStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Detailed
Status</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_summary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Summary</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>List</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_resetTotals">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset
Totals</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_periodicStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Periodic
Status</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_engineSettings">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>220</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Engine Settings</string>
    </property>
    <widget class="QPushButton" name="pushButton_setter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>MIB</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_classMessages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Message
Classes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sim">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Link
Simulator</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>211</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>211</width>
      <height>71</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_autoScroll"/>
    <addaction name="action_logEvents"/>
    <addaction name="action_setBufferSize"/>
    <addaction name="action_clearScreen"/>
    <addaction name="separator"/>
    <addaction name="action_options"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_commands"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_logEvents">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log Events</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_setBufferSize">
   <property name="text">
    <string>Set Buffer Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_clearScreen">
   <property name="text">
    <string>Clear Screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_commands">
   <property name="text">
    <string>Commands</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_options">
   <property name="text">
    <string>Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAuto_Scroll">
   <property name="text">
    <string>Auto-Scroll</string>
   </property>
  </action>
  <action name="action_autoScroll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Scroll</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_input</tabstop>
  <tabstop>plainTextEdit_output</tabstop>
  <tabstop>pushButton_put</tabstop>
  <tabstop>pushButton_suspend</tabstop>
  <tabstop>pushButton_resume</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>pushButton_abandon</tabstop>
  <tabstop>pushButton_report</tabstop>
  <tabstop>pushButton_detailedStatus</tabstop>
  <tabstop>pushButton_periodicStatus</tabstop>
  <tabstop>pushButton_transStatus</tabstop>
  <tabstop>pushButton_list</tabstop>
  <tabstop>pushButton_summary</tabstop>
  <tabstop>pushButton_resetTotals</tabstop>
  <tabstop>pushButton_freeze</tabstop>
  <tabstop>pushButton_disableEngineMessages</tabstop>
  <tabstop>pushButton_setter</tabstop>
  <tabstop>pushButton_classMessages</tabstop>
  <tabstop>pushButton_sim</tabstop>
  <tabstop>pushButton_robust</tabstop>
  <tabstop>pushButton_echo</tabstop>
  <tabstop>pushButton_help</tabstop>
  <tabstop>pushButton_exit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
